layers/vksc_compatibility.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Sam Ricker <sricker@nvidia.com>
+ *
+ * This header is used to provide declarations and defines missing from the standard
+ * VulkanSC Headers such that they can be used in VulkanSC layer builds for compatibility
+ * purposes.
+ */
+#ifndef VKSC_COMPATIBILITY_H
+#define VKSC_COMPATIBILITY_H
+
+typedef VkFlags VkShaderModuleCreateFlags;
+
+typedef struct VkShaderModuleCreateInfo {
+    VkStructureType              sType;
+    const void*                  pNext;
+    VkShaderModuleCreateFlags    flags;
+    size_t                       codeSize;
+    const uint32_t*              pCode;
+} VkShaderModuleCreateInfo;
+
+// These values shouldn't currently overlap with any VulkanSC VkPipelineCreateFlagBits
```

Comment:
You can remove this comment - there will never be overlap between VKSC and Vulkan Flagbits that aren't the same.